src/features/Posts/postsSlice.js

```diff
@@ -3,19 +3,29 @@ import Reddit from "../../util/Reddit";
 
 export const fetchPosts = createAsyncThunk(
 	"posts/fetchPosts",
-	async (nextListing) => {
-		const response = await Reddit.fetchHomePage(nextListing);
+	async (conditions) => {
+		const response = await Reddit.fetchHomePage(conditions);
 		return response;
 	}
 );
 
+export const fetchSearch = createAsyncThunk("posts/fetchSearch", async (searchConditions) => {
```

Comment:
I'm just wondering why you don't use the fetchSearch function in the Nav slice? You use the fetchPosts twice.

Reply:
Oh I forgot to erase that one, Originally I was going to use that one for the search but because everyting's linked, the home page and the infinite scroll I was easier for me to put everything into one function since you have already combined the scroll part with the fetch and added a the after parameter I just added the other two parameters and added the rest of the code